person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    true if @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    true if is_of_age? || parent_permission
+  end
```

Comment:
- [ ] I believe the right method is `of_age?`, not `is_of_age?`. Kindly fix that.

- [ ] You should use the `@parent_permission` variable, instead of `parent_permission` as the latter causes errors.

- [ ] I would recommend you remove the `true` keyword in this method. This is because when neither `of_age?`, nor `@parent_permission` is true, it returns `nil`, whereas it should return `false`

  ```
  def can_use_services?
    of_age? || @parent_permission
  end
  ```